src/schema/search.ts

```diff
@@ -231,7 +231,7 @@ export const typeDefs = /* GraphQL */ `
       """
       Maximum number of users to return
       """
-      limit: Int = ${defaultSearchLimit}
```

Comment:
Would this not be as is and then we just inject 10 if actual limit < 10

Reply:
yeah would also work :) 
Doubt we actually ever want to override the limit though.